FLIR/conservator/generated/generate.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
-
-source ../../../venv/bin/activate
```

Comment:
Not running a Python virtual env any more?

Reply:
I never have. I always disliked that this script assumed that you were running it in from the `FLIR/conservator/generated` directory, and that line reinforces that assumption. 

Also, if you're using a virtual environment, you presumably already have it activated. I guess we could add a check to the script that `sgqlc-codegen` is on the path, and give an error message if not.